### PR TITLE
First draft of the first CoveoBlitz preparation serie

### PR DESCRIPTION
**Pull request by AndyCoveo**

### CODE REVIEW

File: _posts/2017-12-01-blitz-preparation.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+
+The contest was about searches! It was crazy, I knew exactly what I had to do and how to split the project so we could effectively work on this together.
+
+Well, you know, I thought [I knew](https://www.youtube.com/watch?v=cphNpqKpKc4)...
+
+## The Blitz Day
+
+Before I take a deep dive into the madness that is Coveo Blitz, I’d like to take a moment to underline how AWESOME this kind of event is. It’s been six years and I can still remember all those details from the preparation that preceded the day of the event. On my best days, I can barely remember what I had for lunch a day ago. But Coveo Blitz bucks that forgetfulness and is ingrained as an one of those memories you constantly look back on with pride and joy, easily one of my highlights from my college years.. I had the chance to develop friendships, and meet like-minded passionate devs, and got to learn what the heck Coveo is, and truly learn how great the company is.
```
Comment (marcsanfacon) [inline _posts/2017-12-01-blitz-preparation.md]: as an one ->  as one

**Comment by fredboutin** [comment]
Loved it!

File: _posts/2017-12-01-blitz-preparation.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+The contest was about searches! It was crazy, I knew exactly what I had to do and how to split the project so we could effectively work on this together.
+
+Well, you know, I thought [I knew](https://www.youtube.com/watch?v=cphNpqKpKc4)...
+
+## The Blitz Day
+
+Before I take a deep dive into the madness that is Coveo Blitz, I’d like to take a moment to underline how AWESOME this kind of event is. It’s been six years and I can still remember all those details from the preparation that preceded the day of the event. On my best days, I can barely remember what I had for lunch a day ago. But Coveo Blitz bucks that forgetfulness and is ingrained as an one of those memories you constantly look back on with pride and joy, easily one of my highlights from my college years.. I had the chance to develop friendships, and meet like-minded passionate devs, and got to learn what the heck Coveo is, and truly learn how great the company is. 
+
+Stay tuned for the next post - there, I’ll talk about the big day. Maybe you’ll even pick up some interesting tidbits for you and your team. I mean, if you plan on participating. You are participating right?
```
Comment (sbelzile-Coveo) [inline _posts/2017-12-01-blitz-preparation.md]: You are participating right? => You are participating**,** right?

File: _posts/2017-12-01-blitz-preparation.md (hunk 1 above)
Comment (jebeaudet) [inline _posts/2017-12-01-blitz-preparation.md]: from my college years.. (double period)

Comment (jebeaudet) [inline _posts/2017-12-01-blitz-preparation.md]: I had the chance to develop friendships, meet like-minded passionate devs, got to learn what the heck Coveo is, and truly learn how great the company is. 
(Repetition of and + commas)

File: _posts/2017-12-01-blitz-preparation.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Of course, when I asked my friends what they did during their vacation, I had assumed they would have sharpened their programming chops, or at least learned something new. But, to assume is to make an ass out of you and me. Good thing they have some grade-a raw programming skills.
+
+The funny thing is that while this research took place six years ago, I still retain a lot of this information in the ‘ole noggin. Meanwhile, I can barely remember my prolific university courses. I suppose there’s a lesson in there...
+
+## The Blitz Before the Storm
+
+Let me be frank, on the day of the event I was [shipping my pants](https://www.youtube.com/watch?v=2xwUuSM06xQ). I found myself second guessing all the preparation I’ve done. Why did I put all my research eggs in the search engine basket? What if it wasn’t useful at all?? Jesus, the amount of stress I was going through while the Coveo team was announcing the goal of the contest and… OMG yes! That student arrogance of mine payed off!
```
Comment (amoreauCoveo) [inline _posts/2017-12-01-blitz-preparation.md]: change comma to colon

File: _posts/2017-12-01-blitz-preparation.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+
```
Comment (amoreauCoveo) [inline _posts/2017-12-01-blitz-preparation.md]: remove extra line

File: _posts/2017-12-01-blitz-preparation.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+The contest was about searches! It was crazy, I knew exactly what I had to do and how to split the project so we could effectively work on this together.
```
Comment (amoreauCoveo) [inline _posts/2017-12-01-blitz-preparation.md]: about search

**Pull request merged**